.github/workflows/main.yml

```diff
@@ -7,10 +7,10 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
-      - name: Use Node 20
+      - name: Use Node 23
```

Comment:
22 is the latest LTS. Are you sure we want to test with 23?

Reply:
You're right, I'll use 22 then.